packages/ra-input-rich-text/src/index.tsx

```diff
@@ -161,6 +161,7 @@ const RichTextInput: FunctionComponent<Props> = props => {
 };
 
 RichTextInput.propTypes = {
+    // @ts-ignore
```

Comment:
I don't like this fix but haven't found a better one. @fzaninotto ?

Reply:
Nope, don't know any better.